pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -473,14 +472,6 @@ func (l *LibvirtDomainManager) preStartHook(vmi *v1.VirtualMachineInstance, doma
 		return domain, fmt.Errorf("preparing the pod network failed: %v", err)
 	}
 
-	// create disks images on the cluster lever
-	// or initialize disks images for empty PVC
-	hostDiskCreator := hostdisk.NewHostDiskCreator(l.notifier, l.lessPVCSpaceToleration, l.minimumPVCReserveBytes)
-	err = hostDiskCreator.Create(vmi)
-	if err != nil {
-		return domain, fmt.Errorf("preparing host-disks failed: %v", err)
-	}
```

Comment:
if lessPVCSpaceToleration and minimumPVCReserveBytes are only used here, then there's a whole chain of things starting with template.go that can be removed related to these vars.